components/elm/cime_config/config_compsets.xml

```diff
@@ -729,6 +729,11 @@
     <lname>1850_DATM%CPLWCH_ELM%CNPRDCTCBCWFM-CROP_SICE_SOCN_MOSART_SGLC_SWAV</lname>
   </compset>
 
+  <compset>
```

Comment:
A compset with the same name and description (`I20TRWCCNPWFMCROP`) exists on the lines immediately following the current addition. Please delete this new compset.